docker/zcash-lightwalletd/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN go mod download
 COPY . ./
 
 # Build and install the binary.
-RUN go build -v -o /lightwalletd
+RUN go build -v -o /opt/lightwalletd
```

Comment:
After this change, will these entrypoints still work?
Why are they different?

https://github.com/ZcashFoundation/zebra/blob/54e17d40acf79308f8bd0cb79e0545d1b1fca114/docker/zcash-lightwalletd/Dockerfile#L39
https://github.com/ZcashFoundation/zebra/blob/54e17d40acf79308f8bd0cb79e0545d1b1fca114/docker/zcash-lightwalletd/Dockerfile#L60

Reply:
As we do the following `COPY --from=build /opt/lightwalletd /usr/local/bin`

We're, in the end, using `/usr/local/bin` making the `lightwalletd` binary available as a command for execution to the user, just like any other linux command.

I removed the `/ up front` in the final entrypoint, as in this stage `runtime` the binary is only available in `/usr/local/bin` not in the actual `WORKDIR`